Jenkinsfile

```diff
@@ -113,6 +113,9 @@ pipeline{
 		stage('Post: Generate Graph Database'){
 			steps{
 				script{
+					sh "git clone https://github.com/reactome/release-jenkins-utils"
+					sh "mv release-jenkins-utils/scripts/changeGraphDatabase.sh ${env.JENKINS_HOME_PATH}"
```

Comment:
Maybe this should be `cp` to keep the `release-jenkins-utils` repo intact?

Reply:
I thought about it, but I wasn't sure if duplicating a script that has sensitive processes was ok behaviour?

Reply:
Hmm, well it already exists in GitHub and it is being copied to a new location, so I think it is okay.  If you want to ensure the script copy in the new directory is definitely overwritten and up to date, maybe do a `cp -f`?